StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetLPMConfirmFlowTests.swift

```diff
@@ -46,6 +47,30 @@ final class PaymentSheet_LPM_ConfirmFlowTests: XCTestCase {
         await PaymentSheetLoader.loadMiscellaneousSingletons()
     }
 
+    /// 👋 👨‍🏫  Look at this test to understand how to write your own tests in this file
+    func testiDEALConfirmFlows() async throws {
+        try await _testConfirm(intentKinds: [.paymentIntent], currency: "EUR", paymentMethodType: .dynamic("ideal")) { form in
```

Comment:
Side note: Could this make this test work with PMs that use redirects? I tried to use it for Alipay but it failed since it could never load the web view and thus never completed.

Reply:
Yeah it does! Check out https://github.com/stripe/stripe-ios/pull/2906/files/8887046aa5b6af42adaa9436ff94d3f433f0876e#diff-a026e64b444d61822aedf6769b182760b2b8e028a95358d7ccef4cb7dc83c2d6R305-R310 - if we see the redirect we consider it a success.

Reply:
Oh you tried it for Alipay already and it didn't work?  I'm guessing the _redirectShim isn't called, then, maybe because LUXE takes a different path?  It's definitely a brittle approach to use that shim.

Reply:
Hmm, then I wonder why it didn't work for me, I'll try it again